python/tvm/auto_scheduler/measure.py

```diff
@@ -419,7 +419,7 @@ def timed_func():
 
         try:
             sch, args = task.compute_dag.apply_steps_from_state(
-                inp.state)
+                inp.state, layout_rewrite=True)
```

Comment:
Consider to get the `layout_rewrite` parameter from outside? (e.g. from GLOBAL_BUILD_ARGUMENTS)
It's not so good to just set it `True`.

Reply:
Whether to do layout rewrite or not for an op is specified by attr layout_free_placeholders in compute definition, so it's safe to set True by default here.